docs/MODULES.md

```diff
@@ -630,4 +631,4 @@ The modules allows to add text to image in both browser and node environment. We
 ```
 The options can take various attributes like,
 ```
-var options = { text : 'Hello World', size : '12'};
```

Comment:
Any reason for this?

Reply:
No. This should have happened while editing.